README.md

```diff
@@ -1,91 +1,95 @@
-# EVM Placeholder Proof System Verifier 
+# EVM Placeholder proof system verifier 
 
 [![Discord](https://img.shields.io/discord/969303013749579846.svg?logo=discord&style=flat-square)](https://discord.gg/KmTAEjbmM3)
 [![Telegram](https://img.shields.io/badge/Telegram-2CA5E0?style=flat-square&logo=telegram&logoColor=dark)](https://t.me/nilfoundation)
 [![Twitter](https://img.shields.io/twitter/follow/nil_foundation)](https://twitter.com/nil_foundation)
 
-This repository contains the smart contracts for validating zero knowledge proofs 
-generated in placeholder proof system in EVM. 
+This repository contains the smart contracts for validating zero-knowledge proofs generated
+via [Placeholder proof system](https://github.com/NilFoundation/evm-placeholder-verification) in-EVM.
 
 ## Dependencies
 
 - [Hardhat](https://hardhat.org/)
-- [nodejs](https://nodejs.org/en/) >= 16.0
+- [nodejs](https://nodejs.org/en/) - Hardhat requires an LTS version; as of August 2023 it's v18.17.0
```

Comment:
* JFYI: all v18 is a long-term support version (LTS), and 18.17.0 is the current release of LTS.
* Node.js is a proper name; use the correct spelling. 
* Readers have their language settings in the browser. If it's German, the browser will redirect them to https://nodejs.org/de. So, don't include language in the link when it works without it.

```suggestion
- [Node.js](https://nodejs.org/) - Hardhat requires an LTS version; as of September 2023 it's v18.17.1.
```